### ISSUE TITLE
Request count metrics

### ISSUE THREAD

**Issue by SamBarker** [kroxylicious/kroxylicious#107]
As an operator of the proxy I want to be able monitor the amount of ingress and egress traffic to the proxy so that I can monitor usage and determine compliance with usage limits.

To do that the proxy should offer:

- [ ] a count of the number of requests received by a specific frontend 
- [ ] a count of the number of requests sent to a specific backend
- [ ] the total number of bytes of ingress traffic to a specific frontend
- [ ] the total number of bytes of egress traffic from a specific frontend
- [ ] the total number of bytes of ingress traffic from a specific backend
- [ ] the total number of bytes of egress traffic to a specific backend

**Comment by k-wall** [comment]
I think once #139 lands (however it materialises), the metrics should have an address label, which will contain either an upstream or downstream address.

```
kroxylicious_received_bytes_total{flowing=upstream,address="mycluster1-public-bootstrap"}
kroxylicious_received_bytes_total{flowing=upstream,address="mycluster1-public-broker1"} 
kroxylicious_received_bytes_total{flowing=downstream,address="mycluster1-private-bootstrap"} 
kroxylicious_received_bytes_total{flowing=downstream,address="mycluster1-private-broker1"} 
```

In configuration where SNI is not in use, the upstream address would be just the bind address of the listening socket i.e. 0.0.0.0: 9092.

**Comment by robobario** [comment]
closing it as done

**Comment by robobario** [comment]
@k-wall will spin off a new issue for adding some way of identifying the virtual cluster to the metrics

**Comment by k-wall** [comment]
Raised as https://github.com/kroxylicious/kroxylicious/issues/1109
